### ISSUE TITLE
src/stats.c: dead code suspected by coverity

### ISSUE THREAD

**Issue by chipitsine** [haproxy/haproxy#1082]
```
2072                        case ST_F_QCUR:
2073                                break;
    CID 1445766 (#1 of 1): Structurally dead code (UNREACHABLE)unreachable: This code cannot be reached: metric = mkf_u32(0U, sv->nb....
2074                                metric = mkf_u32(0, sv->nbpend);
2075                                break;
2076                        case ST_F_QMAX:
```

**Comment by chipitsine** [comment]
@wdauchy , looks like it might be related to metrics, but I cannot associate with any code change

**Comment by capflam** [comment]
Already fixed by 399bf72f6 and 3888b8cd7 :)

**Comment by wdauchy** [comment]
thanks @capflam !
